src/lfn.c

```diff
@@ -155,6 +147,14 @@ static void copy_lfn_part(unsigned char *dst, LFN_ENT * lfn)
     memcpy(dst + 22, lfn->name11_12, 4);
 }
 
+/* Convert name part in 'lfn' from unicode to ASCII */
+static inline char *cnv_this_part(LFN_ENT *lfn)
+{
+	unsigned char __part_uni[CHARS_PER_LFN*2];
```

Comment:
One thing: prefix `__` is reserved for compiler, so it must not be used for user variables in functions.

Reply:
Done. I also adjusted the whitespace style slightly to match the surrounding code.